### ISSUE TITLE
Transactions history export options (pdf, txt, html)

### ISSUE THREAD

**Issue by serrq** [horizontalsystems/unstoppable-wallet-android#6755]
Some governments or companies may ask for a proof of the real ownership of a title and the related transaction through which you have come to own a certain title.

The export function should be selective, to ensure that only transactions that you wish to forward to a third party actor (a government, an administration, a car dealership, etc.) be effectively shared externally.

This document also include within all the chosen transactions' receipts.

In summary this document produces two types of information (maybe it is better if splitted):

• the records (the overall transactions view)
• receipts (the single transactions with all the details)
